commands/src/main/java/org/wildfly/extras/creaper/commands/infinispan/cache/RemoveCache.java

```diff
@@ -4,11 +4,17 @@
 import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
 import org.wildfly.extras.creaper.core.online.operations.Address;
 import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
 
 public final class RemoveCache implements OnlineCommand {
     private final Address address;
+    private final boolean allowResourceServiceRestart;
 
     public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName) {
+        this(cacheContainer, cacheType, cacheName, false);
+    }
+
+    public RemoveCache(String cacheContainer, CacheType cacheType, String cacheName, boolean allowResourceServiceRestart) {
```

Comment:
Shouldn't we try to have the javadoc upstream related => rather pointing out WildFly releases? Otherwise I am OK with both approaches (Builder one and constructor one with one optional argument.
Regarding having it based on Version seems probably better as user usually just wants to remove the cache and have it working on all EAP versions and not need to modify command attributes based on EAP version it is used with.